README.md

```diff
@@ -15,7 +15,7 @@ This repository contains all the interface definition files for all the CSCs of
 
 * (https://github.com/lsst-ts/ts_sal) 
 
-The SAL produces libraries in C++, LabVIEW, Java, PyDDS, and Python, for each CSC.  These libraries are then used by all the CSCs that need to publish or subscribe to any message topic.
+The SAL produces libraries in C++, LabVIEW, and Java for each CSC.  These libraries are then used by all the CSCs that need to publish or subscribe to any message topic.
 
```

Comment:
This misses ts_salobj, which produces wrappers for Python. Can we somehow include that? Like:

Python object interface is provided by (https://github.com/lsst-ts/ts_salobj).

Reply:
Sure, it's independent but is used to provide Python binding. This chapter is about bindings, and now completely lacks any mention of how Python is interfaced.

Reply:
The Python binding should not be used anymore and will be removed in the future. So it should not be mentioned and anyone using Python needs to use ts_salobj.